pkg/devfile/parser/data/v2/commands.go

```diff
@@ -51,15 +52,19 @@ func (d *DevfileV2) GetCommands(options common.DevfileOptions) ([]v1.Command, er
 // AddCommands adds the slice of Command objects to the Devfile's commands
 // if a command is already defined, error out
 func (d *DevfileV2) AddCommands(commands []v1.Command) error {
-
+	var errorsList []string
 	for _, command := range commands {
 		for _, devfileCommand := range d.Commands {
 			if command.Id == devfileCommand.Id {
-				return &common.FieldAlreadyExistError{Name: command.Id, Field: "command"}
+				errorsList = append(errorsList, (&common.FieldAlreadyExistError{Name: command.Id, Field: "command"}).Error())
+				continue
```

Comment:
IMO we should update teh test to see if all these ids are being returned in the err. Previously we didnt care as it was just return on first error. But since we are updating it for a specific purpose we should probably test it. We can use regex to match err strings like we do in devfile/api 🤔

Reply:
updated